src/main/kotlin/racingcar/Main.kt

```diff
@@ -4,6 +4,6 @@ import racingcar.io.InputView
 import racingcar.racing.RacingGame
 
 fun main() {
-    val input = InputView().getInput()
-    RacingGame(input).start()
+    val gamaParam = InputView().getGameParam()
```

Comment:
```suggestion
    val gameParam = InputView().getGameParam()
```
변수명에 오타가 있는 듯 합니다. 😅